tests/integration/double-reentry.spec.ts

```diff
@@ -19,9 +19,9 @@ describe('Re-entry into queue', () => {
   });
 
   it('Requests are re-entered into queue after the failure of dependency', async () => {
-    assert(failed.validate(prs[branch1].statuses));
-    assert(reentryFail.validate(prs[branch2].statuses));
-    assert(doubleReentrySuccess.validate(prs[branch3].statuses));
+    assert(await validate(prs[branch1].statuses, failed));
+    assert(await validate(prs[branch2].statuses, reentryFail));
+    assert(await validate(prs[branch3].statuses, doubleReentrySuccess));
```

Comment:
Turns out the statuses weren't actually. being verified properly, but it usually didn't matter because these tests just existed to ensure nothing broke on the dev instance of Landkid so a quick manual verification was all that was required.